doc/solver/solver_utility.rst

```diff
@@ -0,0 +1,8 @@
+.. _solver-utility:
+
+Solver utility functions
++++++++++++++++++++++++++++++++++++++
+To be filled later
```

Comment:
List functions and use their docstring for documentation.  If there are no "public" utility functions,  remove the section

Reply:
Done